_data/errors.yml

```diff
@@ -0,0 +1,93 @@
+oidc:
+  auth:
+    - title: Bad client_id
+      id: bad-client-id
+      accordion-id: bad-client
+      content: |
+        ###### Why it's happening
```

Comment:
Can we use h5's on all of these instead of h6's? They are bigger and read more like headers to me visually than h6 which is a little hard to read.

Reply:
This is also important for accessibility, as it looks like H5 would be the correct next level in the heading order.

https://dequeuniversity.com/rules/axe/3.5/heading-order

It'd be good to get some accessibility testing in place since it looks like there's already some existing issues related to this.

Reply:
ah yes sorry, i think this was a typo that got copy/paste propagated, apologies. 

agree with the accessibility testing, although i think that might have to be its own PR depending on how many issues crop up. is there a preferred automated testing tool for login.gov?

Reply:
fixed the headers in https://github.com/18F/identity-dev-docs/pull/382/commits/ec23adf4753192c2c0a61891f6594a25ae604f70